fmriprep/workflows/bold.py

```diff
@@ -381,6 +384,49 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
     return workflow
 
 
+def init_bold_reference_wf(omp_nthreads, bold_file=None, name='bold_reference_wf'):
+    """Generate a reference BOLD image for a series
+
+    This image is the target of head-motion-correction, and the subject of distortion
+    correction and registration to T1w and template spaces.
```

Comment:
Are you planning on adding the workflow generation code within #638 or should we already require ourselves to write the graph whenever we add a new workflow? (even for small ones like this).